client/ayon_core/hosts/substancepainter/api/lib.py

```diff
@@ -11,6 +13,43 @@
 from qtpy import QtGui, QtWidgets, QtCore
 
 
+def get_channel_map_enum():
+    """Function to get channel map items value.
+    For backward compatibility only. Will be removed after
+    client addon migration
+    """
+    return {
+            "BaseColor": "Base Color",
+            "Metallic": "Metallic",
+            "Roughness": "Roughness",
+            "SpecularEdgeColor": "Specular Edge Color",
+            "Emissive": "Emissive",
+            "Opacity": "Opacity",
+            "Displacement": "Displacement",
+            "Glossiness": "Glossiness",
+            "Anisotropylevel": "Anisotropy Level",
+            "AO": "Ambient Occulsion",
+            "Anisotropyangle": "Anisotropy Angle",
+            "Transmissive": "Transmissive",
+            "Reflection": "Reflection",
+            "Diffuse": "Diffuse",
+            "Ior": "Index of Refraction",
+            "Specularlevel": "Specular Level",
+            "BlendingMask": "Blending Mask",
+            "Translucency": "Translucency",
+            "Scattering": "Scattering",
+            "ScatterColor": "Scatter Color",
+            "SheenOpacity": "Sheen Opacity",
+            "SheenRoughness": "Sheen Roughness",
+            "SheenColor": "Sheen Color",
+            "CoatOpacity": "Coat Opacity",
+            "CoatColor": "Coat Color",
+            "CoatRoughness": "Coat Roughness",
+            "CoatSpecularLevel": "Coat Specular Level",
+            "CoatNormal": "Coat Normal",
+    }
```

Comment:
Can we move this to `create_textures.py`? It's only used there - and by moving it there we're hopefully avoiding someone else using this down the line since it's up for removal. Right?